### PR TITLE
LED subsystem update

### PR DESCRIPTION
**Pull request by EliyaFishman**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/led/LED.java [hunk 1]
```diff
@@ -94,6 +98,12 @@ public void setRandomPattern() {
     setControllerPower(getRandomPattern());
   }
 
+  public void startEmergencyLED(){
+    DriverStationLogger.logToDS("Starting emergency LED");
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/subsystems/led/LED.java]: DriverStationLogger.logToDS("Starting emergency LED");
I think you should remove this and in the DriverStationLogger add in the logErrorToDS method:
Robot.led.startEmergencyLED();

AND add stopEmergencyLED method:
`public void stopEmergencyLED() {
    DriverStationLogger.logToDS("LED emergency disabled");
    setColor(LEDColor.Off);
    isInEmergency = false;
}`

File: src/main/java/frc/robot/subsystems/led/LED.java [hunk 2]
```diff
@@ -94,6 +98,12 @@ public void setRandomPattern() {
     setControllerPower(getRandomPattern());
   }
 
+  public void startEmergencyLED(){
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/subsystems/led/LED.java]: public void startEmergencyLED(){
to
  public void startEmergencyLED() {

File: src/main/java/frc/robot/subsystems/led/LEDColor.java [hunk 3]
```diff
@@ -6,16 +6,16 @@
  */
 public enum LEDColor {
     Red(0.61),
+    BlinkingRed(-0.25 ),
```
Comment (HilelAmiel) [inline src/main/java/frc/robot/subsystems/led/LEDColor.java]: BlinkingRed(-0.25 ),
to
    BlinkingRed(-0.25),

**Review by HilelAmiel — CHANGES_REQUESTED** [review]
more things

**Comment by EliyaFishman** [comment]
[LED subsystem update](https://trello.com/c/urx9xkpV/119-led-subsystem-update)

**Review by HilelAmiel — APPROVED** [review]

**Pull request merged**